### PR TITLE
More deprecations

### PR DESCRIPTION
**Pull request by kainino0x**
The deprecation lint is quite incomplete in what it can catch, so for this PR I deleted everything deprecated from the types file, then fixed errors. Still outstanding:

- Removal of GPUFence
- Binding changes

Status: Chromium can't pass these QUITE yet, but soon.

<hr>

**Author checklist for test code/plans:**

- [x] All outstanding work is tracked with "TODO" in a test/file description or `.unimplemented()` on a test.
- [x] New helpers, if any, are documented in `helper_index.md`.
- [ ] (Optional, sometimes not possible.) Tests pass (or partially pass without unexpected issues) in an implementation. (Add any extra details above.)
    
**[Reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md) for test code/plans:** (Note: feel free to pull in other reviewers at any time for any reason.)

- [x] The test path is reasonable, the [description](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) "describes the test, succinctly, but in enough detail that a reader can read only the test plans in a file or directory and evaluate the completeness of the test coverage."
- [x] Tests appear to cover this area completely, except for outstanding TODOs. Validation tests use control cases.
    (This is critical for coverage. Assume anything without a TODO will be forgotten about forever.)
- [x] Existing (or new) test helpers are used where they would reduce complexity.
- [x] TypeScript code is readable and understandable (is unobtrusive, has reasonable type-safety/verbosity/dynamicity).

### CODE REVIEW

**Comment by kainino0x** [comment]
@toji PTAL, and see the reviewer checklist in the PR summary which you should check off.

**Comment by kainino0x** [comment]
Can be reviewed as separate commits

File: src/webgpu/api/operation/rendering/blending.spec.ts [hunk 1]
```diff
@@ -233,15 +229,16 @@ g.test('GPUBlendComponent')
...
+          view: renderTarget.createView(),
           loadValue: dstColor,
+          storeOp: 'store',
         },
       ],
     });
     renderPass.setPipeline(pipeline);
-    if (blendColor) {
+    if (blendConstant) {
       /* eslint-disable-next-line deprecation/deprecation */
```
Comment (toji) [inline src/webgpu/api/operation/rendering/blending.spec.ts:240]: Can this comment be removed now?

File: src/webgpu/api/validation/encoding/cmds/render/dynamic_state.spec.ts [hunk 2]
```diff
@@ -296,7 +297,7 @@ g.test('setBlendColor')
     const { r, g, b, a } = t.params;
     const encoders = t.createDummyRenderPassEncoder();
     /* eslint-disable-next-line deprecation/deprecation */
```
Comment (toji) [inline src/webgpu/api/validation/encoding/cmds/render/dynamic_state.spec.ts:299]: Same here. Can the comment be removed now that we're not using the deprecated function?

**Review by toji — APPROVED** [review]
LGTM, with two questions

File: src/webgpu/api/operation/rendering/blending.spec.ts (hunk 1 above)
Comment (kainino0x) [reply src/webgpu/api/operation/rendering/blending.spec.ts]: Oops, yes. I thought I got all these, evidently not.

**Pull request merged**